### PR TITLE
compile time errors solved, current issue:

### PR DESCRIPTION
**Pull request by wyattgill9**
Failed to send message to consumer 'consumer_1': SendError { .. }
Error: Broker error: Dispatch failed: Failed to send to consumer 'consumer_1': channel closed

### CODE REVIEW

File: .vscode/settings.json
Comment (LYZJU2019) [inline .vscode/settings.json:1]: I think we should not push `.vscode` folder to git.

File: crates/consumer/src/simple_consumer.rs
Comment (LYZJU2019) [inline crates/consumer/src/simple_consumer.rs:1]: Why do we need a `sender` in `Consumer`?

**Review by LYZJU2019 — CHANGES_REQUESTED** [review]
Overall LGTM. Left some comments. Good job!

File: .vscode/settings.json
Comment (wyattgill9) [reply .vscode/settings.json:1]: yes!

**Review by LYZJU2019 — APPROVED** [review]
LGTM

**Pull request merged**